test/acceptance/application-test.js

```diff
@@ -56,18 +56,24 @@ describe('Acceptance | application', function() {
   });
 
   beforeEach(function() {
-    return app.startServer({
-      command: 'fastboot',
-      additionalArguments: ['--serve-assets']
-    });
+    return app.startServer();
   });
 
   afterEach(function() {
     return app.stopServer();
   });
 
   it('works', function() {
-    return request('http://localhost:49741')
+    return request({
+      url: 'http://localhost:49741',
+      headers: {
+        // We have to send the `Accept` header so the ember-cli server sees this as a request to `index.html` and sets
+        // `req.serveUrl`, that ember-cli-fastboot needs in its middleware
+        // See https://github.com/ember-cli/ember-cli/blob/master/lib/tasks/server/middleware/history-support/index.js#L55
```

Comment:
Can you link to commits here instead of master, as the line numbers will gradually drift.